eigen.spec

```diff
@@ -1,14 +1,14 @@
-### RPM external eigen e4c107b451c52c9ab2d7b7fa4194ee35332916ec
+### RPM external eigen b20a61c3a0dc
```

Comment:
better to use original eigen short hash as real version e..g for this PR it shoudl be `d812f411c3f9` . This shows us which eigen commit we started from.

Reply:
I don't understand - then what happens when we push more changes to the same branch ?

Reply:
if we want some reference to the original branching point, I think it would be better to use a date; this way we can easily seen which is the newer (original) version

Reply:
then we only update the `tag` ( https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_1_X/master/eigen.spec#L6 ) to point to the tip of branch. cmsBuild will then build it with version `<eigen-short-hash>-<tagN>`

Reply:
done